conf/locale/ar/LC_MESSAGES/django.po

```diff
@@ -14132,10 +14132,11 @@ msgstr "أهداف تسجيل العضوية لدى {platform_name}"
 #: lms/templates/login.html
 msgid ""
 "Please provide the following information to log into your {platform_name} "
-"account. Required fields are noted by {strong_start}bold text and an "
-"asterisk (*){strong_end}."
+"account. Required fields are noted by <strong class=\"indicator\">bold text and an "
+"asterisk (*)</strong>."
```

Comment:
nit: please revert this. Placeholders `{strong_start}` are used in translation strings instead of HTML tags. Please avoid any manual changes to translation files unless they break things or don't properly render.

Reply:
@zubair-arbi the translation had `<strong>` tags and the key had {strong_start}. Do i need to change the translation and put `{strong_start}` there?

Reply:
![image](https://user-images.githubusercontent.com/20854114/80957303-57b61280-8e1c-11ea-946f-b228260887f4.png)
As you can see in the image, I made this change because in the translation of this string, `<strong>` was used instead of `{strong_start}` as there should be placeholders in the marked key string as well as the actual translation. I should have done it the other way around i guess, i.e. replace the HTML in the translations to placeholder `{strong_start}`. Kindly guide If that's what I should do.

Reply:
![image](https://user-images.githubusercontent.com/20854114/80957624-fa6e9100-8e1c-11ea-87b5-3d1754064b8a.png)
This is from the git repo. This might help explain my point

Reply:
I had a call with @rehan99000 and explained about the logic of placeholders in translation strings.
@rehan99000 we need to follow the correct way.

Reply:
@rehan99000 @asadmanzoor93 We should never ever translate or change values of the placeholders